### ISSUE TITLE
Add link to /reports & update /listen link within detection emails

### ISSUE THREAD

**Issue by scottveirs** [orcasound/orcasite#250]
**Enhancement request**
As a recipient of emails (or other/future) notifications of a new human detection, 
I would like to be able to select a link to the new reports page (in addition to the existing link to listen live), 
so I can quickly replay the annotated recording (out of curiosity, or to validate the listeners classification of the sound).

Currently, the email provides a link for live listening via the web app page for the feed where the detection occurred, but not a link to the reports page: 

<img width="579" alt="Screenshot 2023-10-27 at 11 57 33 AM" src="https://github.com/orcasound/orcasite/assets/14044595/503fd5d3-8243-473a-8195-c1005a317b98">

A minor related suggestion if the link to the reports page is added: distinguish it from the live listening link a bit more by editing the live link line of the email from...

`Listen here:`

to 

`Listen live here:`

**Describe the solution you'd like**

**Step 1:**
**Add a new line with a link to the /reports page** to the email template so that the new email body looks something like this:

```
A new detection has been submitted at Orcasound Lab (orcasound-lab)!
--
Description: Faint fish grunt in vessel noise.
Listeners: 1
Listen live here: https://live.orcasound.net/listen/orcasound-lab
Review report here: https://live.orcasound.net/reports
If you no longer wish to receive these emails, you can unsubscribe here.
```

As of 10/27/23, the link to the report page is [https://beta.orcasound.net/reports](https://beta.orcasound.net/reports)

But after the v3 launch (in mid-November?), the report page link will be [https://live.orcasound.net/reports](https://live.orcasound.net/reports)

**Step 2:**
**Refresh the format of the existing link to the feed page**

As of 10/27/23, the link to the listen page is okay in this format `https://live.orcasound.net/feed-slug`...

...But after the v3 launch (in mid-November?), the listen link will be in this format `https://live.orcasound.net/listen/feed-slug`

**Describe alternatives you've considered**
Anticipating that eventually there may be many reports coming in (e.g. from many listeners annotating audio from many feeds), a longer-term, ideal link would be a unique URL to the sound clip (or candidate group of sound clips) with which the detection was associated.

**Comment by scottveirs** [comment]
@skanderm Some/all of this may already be on your soft launch checklist, but wanted to document the enhancement part at least... ;)

**Comment by skanderm** [comment]
Closed by #255
